app/src/main/java/org/greenstand/android/TreeTracker/view/TextButton.kt

```diff
@@ -121,6 +121,25 @@ fun ApprovalButton(
     }
 }
 
+@Composable
+fun InfoButton(modifier: Modifier = Modifier, isEnabled: Boolean,
+               onClick: () -> Unit,shape: DepthSurfaceShape = DepthSurfaceShape.Circle) {
+    DepthButton(
+
+        modifier = modifier.size(70.dp),
+        isEnabled = isEnabled,
+        colors = AppButtonColors.Default,
+        onClick = onClick,
+        shape = DepthSurfaceShape.Circle,
+        depth = 10f
+    )
+    {
+        Image(painter = painterResource(id = R.drawable.info_icon), modifier = Modifier
+            .size(80.dp),contentDescription =null )
```

Comment:
This shows a white image on a dark button. We should customize the color of the button to match it. The top button color should be the same white color, and the edge "shadow" of the button should be slightly darker.

Reply:
Addressed..!